server/src/chatbot.py

```diff
@@ -32,21 +38,20 @@ def preprocess_input(user_message):
     processed_message = ' '.join(tokens)
     return processed_message
 
-def match_intent_to_output(intent_list):
-    # Loop through each word in the processed message
-    output = []
-    for word in intent_list.split():
-        # Match the word to an intent
-        # For now, we'll just return a placeholder intent
-        if word == 'hello':
-            output.append('greetings')
+def match_intent_to_response(intent):
+
+    file_path = Path(__file__).parent / 'dataset/OIT Responses.xlsx'
+    df = pd.read_excel(file_path, sheet_name="Sheet1")    
```

Comment:
Let's not read from and build the dictionary every time a response is requested, but rather when the chatbot is spun up.